src/pages/api/auth-connect/re-send-identity-validation-code.ts

```diff
@@ -0,0 +1,152 @@
+import {
+  IdentityValueCantBeBlankError,
+  sendIdentityValidationCode,
+  IdentityAlreadyUsedError,
+  IdentityTypes,
+} from "@fewlines/connect-management";
+import { getTracer } from "@fwl/tracing";
+import {
+  Endpoint,
+  HttpStatus,
+  getServerSideCookies,
+  setAlertMessagesCookie,
+} from "@fwl/web";
+import {
+  loggingMiddleware,
+  wrapMiddlewares,
+  tracingMiddleware,
+  errorMiddleware,
+  recoveryMiddleware,
+} from "@fwl/web/dist/middlewares";
+import { NextApiRequest, NextApiResponse } from "next";
+
+import { Handler } from "@src/@types/handler";
+import { UserCookie } from "@src/@types/user-cookie";
+import { insertTemporaryIdentity } from "@src/commands/insert-temporary-identity";
+import { config } from "@src/config";
+import { logger } from "@src/logger";
+import { withAuth } from "@src/middlewares/with-auth";
+import { withSentry } from "@src/middlewares/with-sentry";
+import { getDBUserFromSub } from "@src/queries/get-db-user-from-sub";
+import { getIdentityType } from "@src/utils/get-identity-type";
+
+const handler: Handler = (request, response): Promise<void> => {
+  return getTracer().span(
+    "send-identity-validation-code handler",
```

Comment:
```suggestion
    "re-send-identity-validation-code handler",
```

This could be handy to distinguish the "real" `send-validation-code` handler from the resend one in traces, no ?

Reply:
🦅